.travis.yml

```diff
@@ -2,10 +2,14 @@ language: ruby
 rvm:
   - 2.2.3
 before_script:
+  # Markdown Linter using custom rule style file.
+  - mdl README.md --style _linter/markdown-linter-style.rb
+  - mdl _posts --style _linter/markdown-linter-style.rb
+  - mdl _pages --style _linter/markdown-linter-style.rb
```

Comment:
Yes, your #2 suggestion worked perfectly!  Why didn't I think of that! :+1: 

Fixed in https://github.com/shredtechular/m-lab.github.io/commit/b2d5335c3e6d0e473081d9356bbf12d902864e45